api/tests/unit/features/test_unit_features_views.py

```diff
@@ -2878,7 +3011,7 @@ def test_feature_list_last_modified_values(
         Feature.objects.create(name=f"feature_{i}", project=project)
 
     # When
-    with django_assert_num_queries(16):  # TODO: reduce this number of queries!
+    with django_assert_num_queries(19):  # TODO: reduce this number of queries!
```

Comment:
What are the extra queries we're adding here, and have we confirmed that we don't have any N+1 issues?

Reply:
You're right, I found an N+1 problem. I've fixed it by adding metadata to the prefetch_related used in the queryset.